module-auth/src/test/java/com/inhabas/api/auth/domain/oauth2/cookie/CookieUtilsTest.java

```diff
@@ -1,136 +1,122 @@
-    cookie.setMaxAge(180);
-    request.setCookies(cookie);
+    MockHttpServletRequest request = createRequestWithCookie(COOKIE_NAME, COOKIE_CONTENTS, COOKIE_MAX_AGE);
 
     // when
-    Optional<Cookie> myCookie = CookieUtils.resolveCookie(request, "myCookie");
+    Optional<Cookie> myCookie = CookieUtils.resolveCookie(request, COOKIE_NAME);
 
     // then
-    assertThat(myCookie.isPresent()).isTrue();
-    assertThat(myCookie.get().getValue()).isEqualTo("hello");
-    assertThat(myCookie.get().getMaxAge()).isEqualTo(180);
+    assertThat(myCookie).isPresent()
+            .hasValueSatisfying(cookie -> {
+              assertThat(cookie.getValue()).isEqualTo(COOKIE_CONTENTS);
+              assertThat(cookie.getMaxAge()).isEqualTo(COOKIE_MAX_AGE);
+            });
   }
 
   @DisplayName("response 에 쿠키를 저장한다.")
   @Test
   public void saveCookieToResponse() {
     // given
     MockHttpServletResponse response = new MockHttpServletResponse();
-    String cookieName = "myCookie";
-    String cookieContents = "hello";
 
     // when
-    CookieUtils.setCookie(response, cookieName, cookieContents, 180);
+    CookieUtils.setCookie(response, COOKIE_NAME, COOKIE_CONTENTS, COOKIE_MAX_AGE);
 
     // then
-    Cookie resolvedCookie = response.getCookie(cookieName);
-    assert resolvedCookie != null;
-
-    assertThat(resolvedCookie.getName()).isEqualTo(cookieName);
-    assertThat(resolvedCookie.getValue()).isEqualTo(cookieContents);
-    assertThat(resolvedCookie.getMaxAge()).isEqualTo(180);
+    Cookie resolvedCookie = response.getCookie(COOKIE_NAME);
+    assertThat(resolvedCookie).isNotNull()
+            .satisfies(cookie -> {
+              assertThat(cookie.getName()).isEqualTo(COOKIE_NAME);
+              assertThat(cookie.getValue()).isEqualTo(COOKIE_CONTENTS);
+              assertThat(cookie.getMaxAge()).isEqualTo(COOKIE_MAX_AGE);
+            });
   }
 
   @DisplayName("request 에서 쿠키를 지운다.")
   @Test
   public void removeCookieOfRequest() {
     // given
     MockHttpServletResponse response = new MockHttpServletResponse();
-    MockHttpServletRequest request = new MockHttpServletRequest();
-    Cookie cookie = new Cookie("myCookie", "hello");
-    cookie.setMaxAge(180);
-    request.setCookies(cookie);
+    MockHttpServletRequest request = createRequestWithCookie(COOKIE_NAME, COOKIE_CONTENTS, COOKIE_MAX_AGE);
 
     // when
-    CookieUtils.deleteCookie(request, response, "myCookie");
+    CookieUtils.deleteCookie(request, response, COOKIE_NAME);
 
     // then
-    Cookie deletedCookie = response.getCookie("myCookie");
-    assert deletedCookie != null;
-    assertThat(deletedCookie.getMaxAge()).isEqualTo(0);
-    assertThat(deletedCookie.getValue()).isEqualTo("");
+    Cookie deletedCookie = response.getCookie(COOKIE_NAME);
+    assertThat(deletedCookie).isNotNull()
+            .satisfies(cookie -> {
+              assertThat(cookie.getMaxAge()).isEqualTo(0);
+              assertThat(cookie.getValue()).isEqualTo("");
```

Comment:
```suggestion
              assertThat(cookie.getValue()).isEmpty();
```

빈 문자열인지 확인하는 경우 isEmpty 사용할 수 있습니다.